q2_itsxpress/_itsxpress.py

```diff
@@ -204,12 +109,13 @@ def trim_single(per_sample_sequences: SingleLanePerSampleSingleEndFastqDirFmt,
                 region: str,
                 taxa: str = "F",
                 threads: int = 1,
-                cluster_id: float = default_cluster_id) -> SingleLanePerSampleSingleEndFastqDirFmt:
+                cluster_id: float = default_cluster_id) -> CasavaOneEightSingleLanePerSampleDirFmt:
```

Comment:
`CasavaOneEightSingleLanePerSampleDirFmt` is a more generic container format, accepting both single and paired reads. This affords us some flexibility!